unit-tests/shared/src/test/scala/org/scalanative/testsuite/javalib/lang/ProcessTest.scala

```diff
@@ -330,7 +330,7 @@ class ProcessTest {
     val proc = processSleep(2.0).start()
 
     try {
-      val timeout = 10 * 1000 // Value from Issue 3944
+      val timeout = 30 * 1000 // Value from Issue 3944
```

Comment:
increasing the timeout is probably wise.

Explaining the reasoning for the magic number is
worth retaining but I suggest removing the comment
because it no longer matches the reproducer in
the Issue; broken conceptual link.

Nothing or something like '// empirically determined'

Reply:
i'd keep the comment; it contains a wealth of context, and the value of 10s itself is really well hidden and not as relevant.